tests/BaseTest.php

```diff
@@ -2,23 +2,21 @@
 
 namespace Google\Auth\Tests;
 
-use GuzzleHttp\ClientInterface;
+use Google\Auth\HttpHandler\HttpHandlerFactory;
 use PHPUnit\Framework\TestCase;
 
 abstract class BaseTest extends TestCase
 {
     public function onlyGuzzle6()
```

Comment:
This method needs to be renamed to `atLeastGuzzle6` or `onlyGuzzle6Plus`

Reply:
You're of course right! I didn't change the method at first to make the footprint of my changes as small as possible, but it totally makes sense - done!